components/icon/stories/icon.stories.js

```diff
@@ -169,3 +170,161 @@ const TestTemplate = (args) => html`
 		)}
 	</div>
 `;
+
+/**
+ * Display all icon sizes for the Docs page.
+ */
+const WorkflowSizingTemplate = (args) => html`
+	<div
+		style=${styleMap({
+			"display": "flex",
+			"gap": "24px",
+			"flexWrap": "wrap",
+		})}
+	>
+		${["xs","s","m","l","xl"].map(
```

Comment:
I purposefully left `xxl` out of this for the displayed docs, as there was talk of it being deprecated for Spectrum 2--it's not a size that design includes or has tokens for.